config/Makefile.am

```diff
@@ -343,8 +343,13 @@ UNINSTALL_TARGETS = uninstall-config
 INSTALL_TARGETS += install-modprobe.d
 UNINSTALL_TARGETS += uninstall-modprobe.d
 
+if ENABLE_LOG_TARGET_SYSLOG
+SPACE_QUIRK :=
+OPTARG_LOG_TARGET = --log-target=syslog $(SPACE_QUIRK)
```

Comment:
I don't follow why this space quirk is needed.

Reply:
Sorry. Maybe it's because I already forgot the time with autotools and there would be a better way to implement that. The question is: How can we assign a string ending with a space to a variable. In any other language we would just write:

```
OPTARG_LOG_TARGET = "--log-target=syslog "
```

But not with autotools since the space is a separator. Alternatively I could change this line

```
ExecStart=@sbindir@/firewalld --nofork --nopid @OPTARG_LOG_TARGET@$FIREWALLD_ARGS
```
to

```
ExecStart=@sbindir@/firewalld --nofork --nopid @OPTARG_LOG_TARGET@ $FIREWALLD_ARGS
```

(space between @ and $)

but that would render the service file with two spaces instead of one. Both implementations would work and probably there is a better one without a quirk but I did not find it.

Reply:
Fixed.